src/libmongoc/tests/test-mongoc-client.c

```diff
@@ -3792,6 +3835,31 @@ test_ssl_opts_padding_not_null (void)
 }
 #endif
 
+/* static void */
+/* test_mongoc_bind_ip (void) */
```

Comment:
I will remove this before committing, it's a test that works with a simple test server I wrote to inspect incoming connections.